src/openms/include/OpenMS/FILTERING/ID/IDFilter.h

```diff
@@ -1377,11 +1378,20 @@ namespace OpenMS
 
     /// @name Filter functions for class IdentificationData
     ///@{
-    static void keepBestMatchPerQuery(
+
```

Comment:
Hi Hendrik. Would be great if you could add here more documentation.
We might need to add quite a bit of new IDData code to IDFilter and this one looks super useful. 
Can you add some @brief documentation including the cleanup_affected parameter (e.g., what get's cleaned up)